app/views/hiring_staff/vacancies/review_vacancy_sections/_review_supporting_documents.html.haml

```diff
@@ -9,7 +9,7 @@
 %dl.app-check-your-answers.app-check-your-answers--short
   - if @vacancy.documents.none?
     .app-check-your-answers__contents
-      %dt#supporting_documents{class: "govuk-!-font-weight-bold"}
+      %dt#supporting_documents{class: "govuk-!-font-weight-regular"}
```

Comment:
I think David made a refactoring to be consistent to have a space after and before `{}`

Reply:
Ok